### DISCUSSION TITLE
Lets talk about Package Managers...

### DISCUSSION THREAD

**Discussion by HarshPatel5940** [usebruno/bruno#2753] [Ideas]
Hello folks,
     I am aware that i am not actively in track with the bruno community here. So i had very simple question.
     
What do you think about using [**PNPM**](https://pnpm.io) as package manager as it does not have any negatives and it would significantly speed up the installation speed and also help us in speeding our Github Actions.

I am no pro on how you take these decisions for a such a large product. Do let me know your opinions about this topic

**Comment by jwetzell** [comment]
I'm not seeing a real significant speed increase in a quick test of `npm` vs `pnpm` on this repo but I could be setting it up wrong. I think turning on [package caching](https://github.com/actions/setup-node?tab=readme-ov-file#caching-global-packages-data) would lower the action times although they aren't really slow.

**Reply by HarshPatel5940** [reply]
> I'm not seeing a real significant speed increase in a quick test of `npm` vs `pnpm` on this repo but I could be setting it up wrong. I think turning on [package caching](https://github.com/actions/setup-node?tab=readme-ov-file#caching-global-packages-data) would lower the action times although they aren't really slow.

Hmm, just checked few other resources and it turns out you are true. There's not a lot of difference, when you start caching in actions.
